signbank/dictionary/migrations/0042_new_video_type_choices.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.2.23 on 2024-01-09 05:58
+import random
+
+from django.db import migrations
+from django.conf import settings
+
+
+def get_or_create_tags(apps, schema_validation):
```

Comment:
nitpick: This is a video type field, not a tag

Reply:
oops, good spotting